rules.md

```diff
@@ -3,104 +3,131 @@ layout: default
 title: CODEVID-19 Rules
```

Comment:
I was able to nest the [privacy policy](https://github.com/devedmonton/codevid19.com/blob/master/policies/privacy-policy.html) under a "Policies" folder.  I think it keeps our repo looking tidier if we put these files in there as well.

Reply:
Good call. I'll move these in there.